schema/me/conversation/message.js

```diff
@@ -32,7 +45,25 @@ export const MessageType = new GraphQLObjectType({
     },
     from_email_address: {
       type: GraphQLString,
+      deprecationReason: "Prefer to use the structured `from` field.",
+    },
+
+    from: {
+      type: MessageInitiatorType,
+      resolve: ({ from, from_email_address }) => {
+        const namePartRegex = /"([^"]*)"/
```

Comment:
Eh, Idk! This is basically just parsing the double quotes (`"Name" <email>`), so not sure if we need a module? We can always add one later if we find other cases.